requirements.txt

```diff
@@ -0,0 +1,22 @@
+astroid==2.4.2
```

Comment:
We no longer need to keep this requirements.txt up to date since you've moved over to poetry successfully, we can delete this file completely

Reply:
This has now been removed